editor/components/table-of-contents/style.scss

```diff
@@ -1,8 +1,12 @@
 .table-of-contents__popover.components-popover:not(.is-mobile) .components-popover__content {
 	min-width: 380px;
+	max-height: 100%;
+	overflow-y: auto;
 }
 
 .table-of-contents__popover {
+	height: calc(100% - 85.5px);
```

Comment:
In practice it's not currently an issue since it occupies 0 width, but a bit unnerving to see this cause the element to stretch the full viewport height (i.e. if it did have width, area below the popover content might invisibly intercept mouse interactions):

![image](https://user-images.githubusercontent.com/1779930/34627104-4cef94da-f22d-11e7-8065-a25aed6d39a5.png)